pnpm-workspace.yaml

```diff
@@ -1,3 +1,3 @@
 packages:
-- './@app/*'
-- './packages'
\ No newline at end of file
+  - './@app/*'
+  - './packages/*'
```

Comment:
Each package needs to be its own workspace. The way you'd devised it, all of `./packages` was one workspace. Instead, you want each subdirectory to be its own package.

Reply:
But isn't `./packages` technically a workspace? Or does this `./packages/*` make it so that each new folder inside `./packages` becomes its own package and it shouldn't be treated as a "workspace"?